plugins/toolkit/jetbrains-core/tst/software/aws/toolkits/jetbrains/core/region/AwsRegionProviderTest.kt

```diff
@@ -165,6 +167,13 @@ class AwsRegionProviderTest {
                 override fun resolve(resource: RemoteResource): CompletionStage<Path> = CompletableFuture<Path>().apply {
                     complete(file)
                 }
+
+                // following two methods are not used in this test, make compiler happy
```

Comment:
you can simply just define a default implementation in the interface if this is the generic no-op implementation